lib/manager.js

```diff
@@ -0,0 +1,94 @@
+"use strict";
+
+let Manifest = require("./manifest");
+let createFile = require("./util/files");
+let { generateFingerprint } = require("./util");
+let path = require("path");
+
+module.exports = class AssetManager {
+	constructor(referenceDir, { manifestConfig, fingerprint, exitOnError } = {}) {
+		this.referenceDir = referenceDir;
+		this.fingerprint = fingerprint;
+		this.exitOnError = exitOnError;
+		let manifest = this.resolvePath(manifestConfig.file, { enforceRelative: true });
```

Comment:
Why is this not part of the constructor of manifest?

Reply:
a) I did not want to just pass through an opaque config object (cf. https://github.com/faucet-pipeline/faucet-pipeline/pull/17#discussion_r159902086).
b) `Manifest` doesn't have access to `#resolvePath`.

Reply:
Ok. But then let's go with `manifestPath` instead of `manifest`, ok?